Makefile

```diff
@@ -72,10 +68,20 @@ kubevirt-down:
 kubevirt-sync:
 	scripts/kubevirt.sh sync
 
+.PHONY: cluster-functest
+cluster-functest:
+	source ./scripts/export.sh && go test -v -timeout 0 -count 1 ./tests/functests...
```

Comment:
Can you please drop `export.sh` and use the correct scripts for `cluster` and `kubevirt` targets instead? E.g. just put `KUBECONFIG=$(scripts/kubevirt.sh kubeconfig)` or `KUBECONFIG=$(scripts/kubevirtci.sh kubeconfig)` in front.

Reply:
Done